### PR TITLE
Reduce cluster permissions

### PR DESCRIPTION
**Pull request by qpdpQ**
this pr contains:
1. add v1 APIs back to the bundle (not remove v1a1 yet) 
2. remove extra features
3. Reduce some cluster permissions

### CODE REVIEW

File: bundle/manifests/ibm-cert-manager-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -763,11 +642,11 @@ spec:
   relatedImages:
     - image: icr.io/cpopen/ibm-cert-manager-operator:latest
       name: IBM_CERT_MANAGER_OPERATOR_IMAGE
-    - image: icr.io/cpopen/cpfs/icp-cert-manager-controller:1.10.0-jetstack.1.7.1
+    - image: icr.io/cpopen/cpfs/icp-cert-manager-controller:1.10.0-jetstack.1.9.1
```
Comment (bluzarraga) [inline bundle/manifests/ibm-cert-manager-operator.clusterserviceversion.yaml:645]: cicd just bumped the cert manager version to 1.11.0-jetstack.1.9.1, could you update these to match?

Comment (bluzarraga) [reply bundle/manifests/ibm-cert-manager-operator.clusterserviceversion.yaml:645]: I think there are two other files to update as well

Comment (qpdpQ) [reply bundle/manifests/ibm-cert-manager-operator.clusterserviceversion.yaml:645]: Sure, will update it now

**Comment by qpdpQ** [comment]
for issue https://github.ibm.com/IBMPrivateCloud/roadmap/issues/56056

**Comment by qpdpQ** [comment]
close it, plan to follow these steps: https://github.ibm.com/IBMPrivateCloud/roadmap/issues/56718#issuecomment-52202725

**Pull request closed without merging**